docs/guides/integration-aws-entity-resolution.md

```diff
@@ -0,0 +1,193 @@
+
+- [Create AWS Account](#create-aws-account)
+- [Create UID2 Account](#create-uid2-account)
+- [Subscribe to UID2 on AWS](#subscribe-to-uid2-on-aws)
+
+### Create AWS Account
+
+Sign up for an AWS account, or use an existing one.
+
+For details, see [Getting started: Are you a first-time AWS user?](https://docs.aws.amazon.com/accounts/latest/reference/welcome-first-time-user.html) in the AWS documentation.
+
+### Create UID2 Account
+
+Ask your UID2 contact to register you as a UID2 participant. If you're not sure who to ask, see [Contact Info](../getting-started/gs-account-setup.md#contact-info).
+
+Send your UID2 contact the AWS account ID that you'll use for your UID2 integration on Entity Resolution.
+
+### Subscribe to UID2 on AWS
+
+To participate in UID2 on the Amazon Data Exchange Marketplace, subscribe to the UID2 service. See [Unified ID 2.0 Operator on AWS](https://aws.amazon.com/advertising-marketing/solutions/uid2/).
+
+On receipt of your subscription request, the UID2 team do the following:
+- Review your subscription request.
+- Match your request with the AWS account ID that you sent in [Create UID2 Account](#create-uid2-account).
+- Approve the request.
+
+## Configure AWS Account
+
+UID2 participants store their data in AWS, in an S3 bucket, and integrate with AWS Entity Resolution. The basic steps, once you've created the AWS account (see [Create AWS Account](#create-aws-account)) are as follows.
+
+>NOTE: For detailed instructions, see [Setting up AWS Entity Resolution](https://docs.aws.amazon.com/entityresolution/latest/userguide/setting-up.html) in the AWS documentation.
+
+1. [Create an administrator user](https://docs.aws.amazon.com/entityresolution/latest/userguide/setting-up.html#setting-up-create-iam-user).
+
+2. [Prepare data tables](https://docs.aws.amazon.com/entityresolution/latest/userguide/setting-up.html#prepare-data-tables).
+
+   In AWS Entity Resolution, each input data table contains source records with consumer identifiers. For UID2, the identifier can be either email address or phone number, but there must be only one identifier for each record. The identifier can be matched with other source records that you provide. Each record must have a Unique ID which is defined as the primary key in the schema mapping you create in Step 4. Follow these steps:
+
+   - Save your input data table in a supported data format.
+   - Upload your input data table to Amazon S3.
+   - Create an [AWS Glue](https://docs.aws.amazon.com/glue/latest/dg/what-is-glue.html) table.
+
+3. Create an IAM role for a console user.
+
+4. Create a workflow job role for AWS Entity Resolution.
+
+5. Create schema mapping for the data: see [Creating a schema mapping](https://docs.aws.amazon.com/entityresolution/latest/userguide/create-schema-mapping.html) in the AWS documentation.
+
+## Create a Matching Workflow
+
+When your initial account creation and setup are done, the next step is to create a matching workflow for your data.
```

Comment:
should we mention the instructions below are from The AWS Entity Resolution User Guide https://docs.aws.amazon.com/entityresolution/latest/userguide/create-matching-workflow-provider.html#create-mw-uid

here?

Reply:
@sunnywu we already had a link + reference to the AWS doc, but a little further down. Your link was better. I used yours and moved it up. See update. Thx much.